python/dataset.cpp

```diff
@@ -175,25 +176,30 @@ void bind_data_array_properties(py::class_<T, Ignored...> &c) {
       [](T &self, const VariableConstView &data) { self.data().assign(data); },
       R"(Underlying data item.)");
   c.def_property_readonly(
-      "aligned_coords",
+      "coords",
       py::cpp_function([](T &self) { return self.aligned_coords(); },
                        py::return_value_policy::move, py::keep_alive<0, 1>()),
       R"(
       Dict of aligned coords.)");
```

Comment:
Is it worth defining some kind of seealso since `meta`, `coords`, `attrs` are related.

Reply:
Yes we can do that in the Python wrappers that have been discussed?